src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -71,6 +72,7 @@
         # "guardian_1_addrress_id",
         # "guardian_2_address_id",
         "housing_type",
+        "survey_date",
```

Comment:
You got clarification that date (not year) is appropriate for the household surveys?

Reply:
Yes.